core/src/Revolution/Sources/mysql/modMediaSourceElement.php

```diff
@@ -30,16 +31,16 @@ class modMediaSourceElement extends \MODX\Revolution\Sources\modMediaSourceEleme
                 'default' => 0,
                 'index' => 'pk',
             ),
-            'object_class' => 
+            'object_class' =>
             array (
                 'dbtype' => 'varchar',
                 'precision' => '100',
                 'phptype' => 'string',
                 'null' => false,
-                'default' => 'modTemplateVar',
+                'default' => 'MODX\\Revolution\\modTemplateVar',
```

Comment:
This change needs to be made in the schema files and updated by using the xpdo parse-schema command.

Reply:
https://github.com/modxcms/revolution/blob/3.x/core/model/schema/modx.sources.mysql.schema.xml#L69

Reply:
I'll get this resolved today and post the command so we have it for future reference when updating the model.

Reply:
Here were the commands to update the files once the schemas were updated:

```
core/vendor/bin/xpdo parse-schema --config=core/model/schema/config.php --update=1 --psr4=MODX\\ mysql core/model/schema/modx.sources.mysql.schema.xml core/src/
core/vendor/bin/xpdo parse-schema --config=core/model/schema/config.php --update=1 --psr4=MODX\\ sqlsrv core/model/schema/modx.sources.sqlsrv.schema.xml core/src/
```

Let me know if you have any questions.

Reply:
Done, thanks @Mark-H and @opengeek